src/api/types/contact.ts

```diff
@@ -0,0 +1,13 @@
+export const ContactAwaitingAcceptance = 'awaiting-acceptance'
+export const ContactNotConfirmed = 'not-confirmed'
+export const ContactConfirmed = 'confirmed'
+
+export type ContactStatus = typeof ContactAwaitingAcceptance | typeof ContactNotConfirmed | typeof ContactConfirmed
```

Comment:
Remember to make sure that the same statuses (`strings`) are defined in the backend.